crates/rooch-types/src/transaction/ethereum.rs

```diff
@@ -50,7 +48,9 @@ impl EthereumTransactionData {
             access_list: None,
             max_priority_fee_per_gas: None,
             max_fee_per_gas: None,
-            chain_id: Some(U256::from(RoochChainID::DEV.chain_id().id())),
+            chain_id: Some(U256::from(
+                RoochMultiChainID::Rooch.multichain_id().id(), // TODO use Ethereum multichain id and build a multichain genesis init
```

Comment:
This chain id is not the multichain id

Reply:
Currently I changed to `20230103` in default genesis build. Hopefully I will build ethereum chain id on genesis init later.